### ISSUE TITLE
PHP Error when scanning photos

### ISSUE THREAD

**Issue by pannal** [nextcloud/maps#225]
I'm encountering this extremely often:

```
A non-numeric value encountered at /var/www/html/custom_apps/maps/vendor/lsolesen/pel/src/PelIfd.php#370
```

**Comment by jancborchardt** [comment]
Do you have any more info where it could come from? Any more related server logs, or if it comes from pictures, or contacts? cc @tacruc

**Comment by wehrstedt** [comment]
Found the same error but in different line:
```
[PHP] Error: A non-numeric value encountered at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelIfd.php#240

at 2019-12-14T22:06:08+00:00[PHP] Error: A non-numeric value encountered at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelIfd.php#240

at 2019-12-14T22:06:08+00:00
```

**Comment by pannal** [comment]
It comes from pictures, definitely. I've recently added a whole bunch of JPEGs that I wanted to be able to see on the map.

**Comment by tacruc** [comment]
@pannal Which version of the app are you using? 
@wehrstedt  Can you provide the full log entry? There should be some more lines in the stacktrace?

**Comment by wehrstedt** [comment]
Sorry, i flushed the logs because there were hundreds of entries... But that was all I saw in the log viewer. I copied the message via the copy Button in the log viewer in the administration. Is there a command to trigger a rescan? I installed the extension yesterday, so it has to be from Image scanning, didnt used the extension manually.

**Comment by pannal** [comment]
> @pannal Which version of the app are you using?

The newest one from the stable NC branch (0.1.2).

**Comment by simonspa** [comment]
I'm on Maps 0.1.6 with Nextcloud 17.0.4 and see the same errors:

```
{"reqId":"4A0R2LOhf00HZhMM08NU","level":3,"time":"2020-03-14T09:47:37+01:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"A non-numeric value encountered at \/var\/www\/nextcloud\/apps\/maps\/vendor\/lsolesen\/pel\/src\/PelIfd.php#447","userAgent":"--","version":"17.0.4.2"}
```

unfortunately this is the full log entry, nothing more.

**Comment by tacruc** [comment]
It's an error thrown by the exif libery if it thinks that the exif of the image is broken. I'm not shure if we can catch it.

**Comment by tacruc** [comment]
Should be fixed in v0.2.5 nightly
